src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -333,11 +333,12 @@ public async Task<(string? error, Exception? exception)> ExpectCircuitErrorAndDi
             return null;
         }
 
-        public async Task<bool> ConnectAsync(Uri uri, bool connectAutomatically = true)
+        public async Task<bool> ConnectAsync(Uri uri, Action<HubConnectionBuilder, Uri>? configure = null, bool connectAutomatically = true)
```

Comment:
suggest putting the `Action<>` parameter last or else this will be pretty wierd.